syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -687,6 +687,12 @@ def eq(self, _self, other):
 
     __eq__ = eq
 
+    def var(self, **kwargs):
+        # Note that this implem is different from pytorch (which divides by m-1)
+        mu = self.mean(**kwargs)
+        unbiased_self = self - mu
+        return (unbiased_self * unbiased_self).mean(**kwargs)
```

Comment:
:+1: 

This seems to be the correct way of "variance"
What does torch do different?